pkg/zng/docs/spec.md

```diff
@@ -247,16 +540,47 @@ of a value:
 In addition, `-` must be escaped if representing the single ASCII byte equal
 to `-` as opposed to representing an unset value.
 
+## Value Syntax
+
+Each UTF-string field parsed from a value line is interpreted according to the
+type descriptor of the line.
+The formats for each type is as follows:
+
+Type | Format
+---- | ------
+`bool` | a single characeter `T` or `F`
+`byte` | two-characters of hexadecimal digit
+`int16` | decimal string representation of any signed, 16-bit integer
+`uint16` | decimal string representation of any unsigned, 16-bit integer
+`int32` | decimal string representation of any signed, 32-bit integer
+`uint32` | decimal string representation of any unsigned, 32-bit integer
+`int64` | decimal string representation of any signed, 64-bit integer
+`uint64` | decimal string representation of any unsigned, 64-bit integer
+`float64` | a decimal representation of a 64-bit IEEE floating point literal as defined in JavaScript
+`string` | a UTF-8 string
+`bytes` | a sequence of bytes encoded as base64
+`bstring` | a UTF-8 string with \x escapes of non-UTF binary data
+`enum` | a string representing an enumeration value defined outside the scope of ZNG
+`ip` | a string representing an IP address in [IPv4 or IPv6 format](https://tools.ietf.org/html/draft-main-ipaddr-text-rep-02#section-3)
+`net` | a string in CIDR notation representing an IP address and prefix length as defined in RFC 4632 and RFC 4291.
+`time` | unsigned dotted decimal notation of seconds (32-bit second, 32-bit nanosecond)
```

Comment:
what is `32-bit second` conveying here - that seconds must be <= 4294967296?
I don't think `32-bit nanosecond` is needed?

Reply:
Yeah his is a bit off.

Reply:
fixed